components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -2084,7 +2084,7 @@ def generateArray(self, array, name):
             return ""
 
         def specData(const):
-            return (str_to_const_array(const.identifier.name),
+            return (str_to_byte_slice(const.identifier.name),
                     convertConstIDLValueToJSVal(const.value))
```

Comment:
I traced usage of this to https://github.com/abumsoo/servo/blob/8e9807fa6f3cfb82c765002d480e9d2da8f1ea4a/components/script/dom/bindings/codegen/CodegenRust.py#L1636, so this would require change in https://github.com/servo/servo/blob/1c6b74e1f16f6c92f80410d98608064fa4fbede2/components/script/dom/bindings/constant.rs#L16-L21

Reply:
I saw your answer below for your process and it made a lot of sense. However I'm having  a hard time connecting the dots on this particular one. How did you go from https://github.com/abumsoo/servo/blob/8e9807fa6f3cfb82c765002d480e9d2da8f1ea4a/components/script/dom/bindings/codegen/CodegenRust.py#L1636 to 
https://github.com/servo/servo/blob/1c6b74e1f16f6c92f80410d98608064fa4fbede2/components/script/dom/bindings/constant.rs#L16-L21
I'm not seeing anything in `generateGuardedArray` that would make me think to check inside of `constant.rs`.
The only thing I would have thought of would be to search for files where `ConstantSpec` is being used.

Reply:
https://github.com/abumsoo/servo/blob/8e9807fa6f3cfb82c765002d480e9d2da8f1ea4a/components/script/dom/bindings/codegen/CodegenRust.py#L2086 is used here: https://github.com/abumsoo/servo/blob/8e9807fa6f3cfb82c765002d480e9d2da8f1ea4a/components/script/dom/bindings/codegen/CodegenRust.py#L2095 (passed as "fn pointer" or whatever this is called in python). And looking other arguments, and how they are used inside `generateGuardedArray`, it appears to me that `ConstantSpec` needs to be fixed. Then I searched for where `ConstantSpec` is defined.

Reply:
This one was very tricky to be honest.

Reply:
Ohh I see. Thank you for walking me through your thought process. That was very helpful.